backends/dpdk/options.h

```diff
@@ -45,6 +54,13 @@ class DpdkOptions : public CompilerOptions {
         registerOption("-o", "outfile",
                 [this](const char* arg) { outputFile = arg; return true; },
                 "Write output to outfile");
+        registerOption("--context", "file",
+                [this](const char *arg) { ctxtFile = arg; return true; },
+                "Generate and write context JSON to the specified file");
+        registerOption("--testcontextJson", nullptr,
```

Comment:
If we remove the context json output comparison from regressions, this option will no longer be needed.

BTW, I need to avoid printing some fields based on the flag for this options. Would it be possible to achieve that without introducing an option?
https://github.com/p4lang/p4c/blob/4b5f7b43b17d6a6db7a994e2f41a4f6177a6231e/backends/dpdk/dpdkContext.cpp#L120